internal/query/projection/system_features.go

```diff
@@ -0,0 +1,98 @@
+package projection
+
+import (
+	"context"
+
+	"github.com/zitadel/zitadel/internal/eventstore"
+	old_handler "github.com/zitadel/zitadel/internal/eventstore/handler"
+	"github.com/zitadel/zitadel/internal/eventstore/handler/v2"
+	"github.com/zitadel/zitadel/internal/repository/feature/feature_v2"
+	"github.com/zitadel/zitadel/internal/zerrors"
+)
+
+const (
+	SystemFeatureTable = "projections.system_features"
+
+	SystemFeatureKeyCol          = "key"
+	SystemFeatureCreationDateCol = "creation_date"
+	SystemFeatureChangeDateCol   = "change_date"
+	SystemFeatureSequenceCol     = "sequence"
+	SystemFeatureValueCol        = "value"
+)
+
+type systemFeatureProjection struct{}
+
+func newSystemFeatureProjection(ctx context.Context, config handler.Config) *handler.Handler {
+	return handler.NewHandler(ctx, &config, new(systemFeatureProjection))
+}
+
+func (*systemFeatureProjection) Name() string {
+	return SystemFeatureTable
+}
+
+func (*systemFeatureProjection) Init() *old_handler.Check {
+	return handler.NewTableCheck(handler.NewTable(
+		[]*handler.InitColumn{
+			handler.NewColumn(SystemFeatureKeyCol, handler.ColumnTypeText),
+			handler.NewColumn(SystemFeatureCreationDateCol, handler.ColumnTypeTimestamp),
+			handler.NewColumn(SystemFeatureChangeDateCol, handler.ColumnTypeTimestamp),
+			handler.NewColumn(SystemFeatureSequenceCol, handler.ColumnTypeInt64),
+			handler.NewColumn(SystemFeatureValueCol, handler.ColumnTypeJSONB),
+		},
+		handler.NewPrimaryKey(SystemFeatureKeyCol),
+	))
+}
+
+func (*systemFeatureProjection) Reducers() []handler.AggregateReducer {
+	return []handler.AggregateReducer{{
+		Aggregate: feature_v2.AggregateType,
+		EventReducers: []handler.EventReducer{
+			{
+				Event:  feature_v2.SystemResetEventType,
+				Reduce: reduceInstanceResetFeatures,
+			},
+			{
+				Event:  feature_v2.SystemLoginDefaultOrgEventType,
+				Reduce: reduceInstanceSetFeature[bool],
+			},
+			{
+				Event:  feature_v2.SystemTriggerIntrospectionProjectionsEventType,
+				Reduce: reduceInstanceSetFeature[bool],
+			},
+			{
+				Event:  feature_v2.SystemLegacyIntrospectionEventType,
+				Reduce: reduceInstanceSetFeature[bool],
+			},
+		},
+	}}
+}
+
+func reduceSystemSetFeature[T any](event eventstore.Event) (*handler.Statement, error) {
+	e, ok := event.(*feature_v2.SetEvent[T])
```

Comment:
While testing locally, the projection wasn't updated and the breakpoint here didn't hit.

Reply:
fixed, I used the intance reducers in the code above. Copy-paste error.